Sources/Sentry/ThreadInfoHelper.mm

```diff
@@ -0,0 +1,17 @@
+#import "ThreadInfoHelper.h"
+#import "SentryThread.h"
+#include "SentryThreadHandle.hpp"
+
+#if SENTRY_TARGET_PROFILING_SUPPORTED
+
+@implementation ThreadInfoHelper
```

Comment:
`m`: We should prefix SDK classes with `Sentry`, because the prefix is used in our SDK Crash Detection to identify frames of the SDK.

```suggestion
@implementation SentryThreadInfoHelper
```